### ISSUE TITLE
Neural Enhanced Belief Propagation for Cooperative Localization. (arXiv:2105.12903v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#7350]
Neural Enhanced Belief Propagation for Cooperative Localization. (arXiv:2105.12903v1 [cs.LG])<br>
https://ift.tt/3fpMUdp<br>
<p>Location-aware networks will introduce innovative services and applications for modern convenience, applied ocean sciences, and public safety. In this paper, we establish a hybrid method for model-based and data-driven inference. We consider a cooperative localization (CL) scenario where the mobile agents in a wireless network aim to localize themselves by performing pairwise observations with other agents and by exchanging location information. A traditional method for distributed CL in large agent networks is belief propagation (BP) which is completely model-based and is known to suffer from providing inconsistent (overconfident) estimates. The proposed approach addresses these limitations by complementing BP with learned information provided by a graph neural network (GNN). We demonstrate numerically that our method can improve estimation accuracy and avoid overconfident beliefs, while its computational complexity remains comparable to BP. Notably, more consistent beliefs are obtained by not explicitly addressing overconfidence in the loss function used for training of the GNN.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
